src/components/UI.css

```diff
@@ -0,0 +1,53 @@
+.calculator {
+  width: 280px;
+  margin: 0 auto;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.result {
+  background-color: #f2f2f2;
+  border: 1px solid #ccc;
+  border-radius: 3px;
+  padding: 5px;
+  margin-bottom: 10px;
+  text-align: right;
+  font-size: 18px;
+}
+
+.buttons {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 5px;
+}
+
+button {
+  padding: 10px;
+  font-size: 18px;
+  text-align: center;
+  background-color: #e6e6e6;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.button-zero {
+  grid-column: span 2;
+}
+
+.button-equals {
+  grid-column: span 2;
+}
+
+button.operator {
+  background-color: #ffc266;
+}
+
+button:hover {
+  background-color: #d9d9d9;
+}
+
+button:active {
+  background-color: #bfbfbf;
+}
```

Comment:
Nice job so far👍. Your calculator design looks really beautiful😉. But for the sake of meeting the requirements of this project, let's try to stick to the given design😉. when we are done with the entire project you can further add styles to your taste to make it look better😉. But for now, let's stick with the design🙏.

<img width="604" alt="image" src="https://github.com/Zafron047/math-magician/assets/64643995/eadd0a07-975f-45d7-8d28-47ec777ff113">


your design
<img width="346" alt="image" src="https://github.com/Zafron047/math-magician/assets/64643995/4fa8d4de-d248-4046-b1eb-b5c5746c3545">

Reply:
@Ibizugbe Thanks a lot for your kind review. 

- [x] I have made removed the index.css file.
- [x] I have tried best to restore styling as the original design.

Best,

@Ibizugbe